Makefile

```diff
@@ -0,0 +1,53 @@
+.PHONE: vm-kernel
+vm-kernel:
+	cd neonvm && make kernel
+
+.PHONE: vm-local-cluster
+vm-local-cluster:
+	cd neonvm && make local-cluster
+
+.PHONY: vm-deploy
+vm-deploy:
+	cd neonvm && make deploy
+
+AUTOSCALER_SCHEDULER_IMG ?= kube-autoscale-scheduler:dev
+AUTOSCALER_AGENT_IMG ?= autoscaler-agent:dev
+VM_INFORMANT_IMG ?= vm-informant:dev
+
+.PHONY: docker-build
+docker-build:
+	docker buildx build \
+		-t $(AUTOSCALER_SCHEDULER_IMG) \
+		--build-arg "GIT_INFO=n/a" \
+		-f build/autoscale-scheduler/Dockerfile \
+		.
+	docker buildx build \
+		-t $(AUTOSCALER_AGENT_IMG) \
+		--build-arg "GIT_INFO=n/a" \
+		-f build/autoscaler-agent/Dockerfile \
+		.
+	docker buildx build  \
+		-t $(VM_INFORMANT_IMG) \
+		--build-arg "GIT_INFO=n/a" \
+		-f build/vm-informant/Dockerfile \
+		.
+
+.PHONY: kind-load
+kind-load: docker-build
+	kind load docker-image $(AUTOSCALER_SCHEDULER_IMG)
+	kind load docker-image $(AUTOSCALER_AGENT_IMG)
+	kind load docker-image $(VM_INFORMANT_IMG)
+
+.PHONE: deploy
+deploy: kind-load
+
+EXAMPLE_VM_IMG ?= vm-example:dev
```

Comment:
I'd put all the variables near the top, just for readability (make always has  to parse/run global stuff anyway, so you don't get much by putting things at the bottom.